docs/core/toc.yml

```diff
@@ -54,27 +54,33 @@
       href: tutorials/cli-templates-create-template-pack.md
   - name: Get started with .NET Core on macOS
     href: tutorials/using-on-macos.md
-  - name: Get started with .NET Core on macOS using Visual Studio for Mac
+  - name: Get started with .NET Core using Visual Studio for Mac
     href: tutorials/using-on-mac-vs.md
-  - name: Building a complete .NET Core solution on macOS using Visual Studio for Mac
+  - name: Build a complete .NET Core solution on macOS
     href: tutorials/using-on-mac-vs-full-solution.md
   - name: Get started with .NET Core using the CLI tools
     href: tutorials/using-with-xplat-cli.md
-  - name: Organizing and testing projects with the .NET Core command line
+  - name: Organize and test projects with the .NET Core CLI
     href: tutorials/testing-with-cli.md
-  - name: Developing Libraries with Cross Platform Tools
+  - name: Develop libraries with cross-platform tools
     href: tutorials/libraries.md
-  - name: Create a .NET Core application with plugins
-    href: tutorials/creating-app-with-plugin-support.md      
-  - name: Developing ASP.NET Core applications
+  - name: Create a .NET Core app with plugins
+    href: tutorials/creating-app-with-plugin-support.md
+  - name: Develop ASP.NET Core apps
     href: tutorials/aspnet-core.md
-  - name: Hosting .NET Core from native code
+  - name: Host .NET Core from native code
     href: tutorials/netcore-hosting.md
-- name: Native Interoperability
+- name: Run-time configuration
   items:
-  - name: Exposing .NET Core Components to COM
+  - name: Settings
```

Comment:
add the new section to the breadcrumb file

Reply:
It looks like only the Tools subdirectory currently has an entry in the [breadcrumb](https://github.com/dotnet/docs/blob/master/docs/breadcrumb/toc.yml) file. Is there a plan to add all the other subdirectories too?

Reply:
Probably an oversight. Can you file an issue for us to review that?